Gulpfile.js

```diff
@@ -148,7 +163,7 @@ gulp.task('clean-lib', function (cb) {
     del('lib', cb);
 });
 
-gulp.task('transpile-lib', ['lint', 'clean-lib'], function () {
+gulp.task('transpile-lib', ['clean-lib'], function () {
```

Comment:
As i see there is no any task that depends on the `lint` task, as a result you don't check the code during building